downstream/modules/vault-aap/con-vault-migration-examples-kv2-secret-module.adoc

```diff
@@ -32,8 +32,8 @@ After (`hashicorp.vault`):
       foo: bar
 ----
 
```

Comment:
both kv1 and kv2 delete operations are `soft-delete`, probably would be good to put a note somewhere at the start of doc to avoid user confusion on delete. https://developer.hashicorp.com/vault/docs/secrets/kv/kv-v2/cookbook/delete-data

Reply:
Ok, just mentioned it for KV2 only, per Hashi doc link.